src/components/button/Button.js

```diff
@@ -0,0 +1,12 @@
+function Button({description, consoleLog, disabled}) {
+    return (<>
+        <button onClick={() => console.log({consoleLog})}
```

Comment:
Ik zou hier een type="button" toevoegen. Nu voeg je het elke keer toe in je app.js. Scheelt weer tik werk.